TG-Netflix/src/styles/BannerStyles.jsx

```diff
@@ -0,0 +1,479 @@
+    background-position: center center;
+    animation: cssAnimation .25s ease-out 2s forwards, cssAnimation2 .25s ease-out 10s forwards;
+    }
+
+    .overlay {
+        width: 100vw;
+        height: 80vh;
+        position: absolute;
+        display: flex;
+        justify-content: space-between;
+        align-items: flex-end;
+    }
+
+    .banner-contents {
+    position: relative;
+    color: #fff;
+    margin-left: 30px;
+    padding-top:  140px;
+    }
+
+    .banner-title {
+    font-size: 3rem;
+    font-weight: 800;
+    padding-bottom: 0.3rem;
+    max-width: 500px;
+    animation: shrink .5s ease-in-out 5s forwards;
+    }
+    @keyframes shrink {
+        from {
+            transform: scale(1);
+        }
+        to {
+            transform: translate(-15%, 50%) scale(0.65);
+        }
+    }
+
+    .banner-description {
+    width: 45rem;
+    height: 5.438rem;
+    line-height: 1.3;
+    padding-top: 1rem;
+    font-size: 1.44rem;
+    text-shadow: rgba(0,0,0 0.45) 0.125rem 0.125rem 0.25rem;
+    animation: cssAnimation .25s ease-out 5s forwards;
+    }
+
+    @keyframes cssAnimation {
+        from {
+            opacity: 1;
+        }
+        to {
+            opacity: 0;
+        }
+    }
+    @keyframes cssAnimation2 {
+        from {
+            opacity: 0;
+        }
+        to {
+            opacity: 1;
+        }
+    }
+
+    .button-container {
+        width: 75%;
+        height: 72px;
+        display: flex;
+        font-size: 14.4px;
+        line-height: 12.672px;
+        margin: 28.8px 0px 0px;
+    }
+    .play-button-container {
+        align-items: center;
+        color: rgba(255, 255, 255, 0.9);
+        display: flex;
+        font-size: 14.4px;
+        justify-content: center;
+        line-height: 12.672px;
+    }
+    .play-button {
+        width: 208px;
+        height: 58px;
+        align-items: center;
+        background-color: rgba(255, 255, 255, 0.75);
+        border: 0;
+        border-radius: 4px;
+        display: flex;
+        font-size: 14.4px;
+        justify-content: center;
+        line-height: 12.672px;
+        margin: 0px 14.4px 14.4px 0px;
+        padding: 11.52px 34.56px 11.52px  28.8px;
+        text-align: center;
+        cursor: pointer;
+    }
+    .play-button:hover {
+        background-color: rgba(200, 200, 200, 0.75);
+    }
+
+    .meer-info-button {
+        width: 290px;
+        height: 58px;
+        align-items: center;
+        background-color: rgba( 109, 109, 110, 0.4);
+        border: 0;
+        border-radius: 4px;
+        color: rgba(255, 255, 255, 1);
+        display: flex;
+        font-size: 14.4px;
+        justify-content: center;
+        line-height: 12.672px;
+        margin: 0px 14.4px 14.4px 0px;
+        padding: 11.52px 34.56px 11.52px 28.8px;
+        text-align: center;
+        cursor: pointer;
+    }
+    .meer-info-button:hover {
+        background-color: rgba( 109, 109, 110, 0.2);
+    }
+
+    .Hawkins-icon {
```

Comment:
je commit is van vorige week dus wellicht is dit al opgelost.. maar ik zie de icons heel erg klein en drie keer, de replay/mute werkt daardoor ook niet.